uaclient/messages/__init__.py

```diff
@@ -81,6 +81,11 @@ class TxtColor:
     "Finished upgrade of Ubuntu Pro service configuration"
 )
 
+PRO_ONLY_ALLOWED_FOR_RELEASE = t.gettext(
+    "Detaching Ubuntu Pro. Previously attached subscription \
+was only valid for Ubuntu {release_codename} {release} release."
```

Comment:
a) same standardization comment
b) should we have an acceptance test with this flow?